examples/src/main/kotlin/usageExamples/update/Update.kt

```diff
@@ -0,0 +1,55 @@
+package usageExamples.update
+
+// :replace-start: {
+//    "terms": {
+//       "CONNECTION_URI_PLACEHOLDER": "\"<connection string uri>\"",
+//       "import io.github.cdimascio.dotenv.dotenv\n": ""
+//    }
+// }
+// :snippet-start: update-usage-example
+
+import com.mongodb.MongoException
+import com.mongodb.client.model.Filters
+import com.mongodb.client.model.UpdateOptions
+import com.mongodb.client.model.Updates
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.coroutines.runBlocking
+import java.util.*
+
+data class Movie(val title: String, val runtime: Int, val genres: List<String>, val lastUpdated: Date)
```

Comment:
Ben: if you query with Movie, you get wonky data decoding errors (tried with Timestamp, BsonTimestamp, BsonDateTime, etc. without success). Querying by Document or another data class works, but I assume you can also solve with a codec?

Reply:
yeah it's b/c you're using different date data types in the data class and the update operation. in the data class you use `Date`, in the update opp you use `Timestamp`

will poke around to see what a good solution is here